guidelines/content-guidelines/release-notes.md

```diff
@@ -8,7 +8,7 @@ Information in release notes must provide readers with everything they need to k
 * Does an error message appear?
 * Was the enhancement based upon customer feedback?
 
-Because the release notes contain critical information and act as an important communication tool, follow these guidelines so that the documentation is informative and consistent. When authoring release notes, follow the [Style and Terminology](./style-and-terminology.md) for many agreed-upon standards.
+Because the release notes contain critical information and act as an important communication tool, follow these guidelines so that the documentation is informative and consistent. When authoring release notes, follow the [Style and Terminology](./style-and-terminology.md) for many agreed-upon standards. Read [this](https://github.com/kyma-project/website/blob/master/docs/write-blog-posts.md) document to learn how to add the release notes as a blog post on a website.
```

Comment:
```suggestion
Because the release notes contain critical information and act as an important communication tool, follow these guidelines so that the documentation is informative and consistent. When authoring release notes, follow the [Style and Terminology](./style-and-terminology.md) for many agreed-upon standards. Read [this](https://github.com/kyma-project/website/blob/master/docs/write-blog-posts.md) document to learn how to add release notes as a blog post on a website.
```

Reply:
But I would add this reference in the linking doc - I guess this is the first reference when you search for instructions on adding links. What do you think?